bigtable/google/cloud/bigtable/client.py

```diff
@@ -100,6 +102,13 @@ class Client(ClientWithProject):
                   interact with the Instance Admin or Table Admin APIs. This
                   requires the :const:`ADMIN_SCOPE`. Defaults to :data:`False`.
 
+    :type: client_info: :class:`google.api_core.client_info.ClientInfo`
```

Comment:
Does this need to be the `google.api_core.gapic_v1.client_info.ClientInfo` version, instead?

What happens when a `google.api_core.client_info.ClientInfo` is used? I imagine it might fail to populate the `x-goog-api-client` header?